site/zh/blog/new-in-devtools-105/index.md

```diff
@@ -29,169 +28,177 @@ draft: true
 
```

Comment:
item 5, please update the zh whats-new.md file as well.

Reply:
where can i find the whats-new.md file?

Reply:
You can find it with this path: https://github.com/GoogleChrome/developer.chrome.com/tree/main/site/_includes/partials/devtools/zh